CHANGELOG.rst

```diff
@@ -16,6 +16,13 @@ Change Log
 Unreleased
 ----------
 
+* Nothing.
```

Comment:
Lol, "nothing" ;) Is this "Unreleased" section of the changelog even necessary? Is it intended to be for things that were merged but not yet released to PyPi?

Reply:
Yeah, exactly.  I think it's suggested to have one by the semantic versioning specification.

I added "nothing" here because it's always a tiny bit confusing for my eyes to understand
```
Unreleased
-----------
[x.y.z]
----------------
* Is this where I put _my_ change?
```